helpers.sh

```diff
@@ -498,18 +509,31 @@ export_coveralls_data() {
 
   ensure_jq_binary # required for git_log
 
+
+  if ! git --no-pager log -1 > /dev/null 2>&1; then
```

Comment:
FYI: tried to use `git_log` here but that doesn't report the exit code, even with `set -o pipefail`.

Reply:
If we leave the check like this, no. But if I wanted to write `if ! git_log "%ae"; then...` it wouldn't work. That's why this is an extra line instead of checking the first use of `git_log` below.